hooks/useLocaleUrl.ts

```diff
@@ -0,0 +1,16 @@
+'use client'
+
+export function generateNewUrl(pathname: string, language: string): string {
+  const pathnameWithoutLanguage = pathname.replace(/^\/(en|nl)\b/, '')
```

Comment:
Does this function only search for `en` or `nl` in the url? if so we might want to do some future proofing for any translations in the future

Reply:
yes, currently this regular expression is only looking for en and nl.

Reply:
`.replace(/^\/..\b/, '')`
Would this replace() work to replace any future language code?